addons/ctrl-enter-post/comments.js

```diff
@@ -0,0 +1,22 @@
+export default async function ({ addon, global, console, msg }) {
+  let isScratchR2 = addon.tab.clientVersion == "scratchr2";
+
+  let textboxSelector = isScratchR2 ? "[name='content']" : "[name='compose-comment']";
+  // TODO: fix on user profiles as it fails
```

Comment:
is this still a problem?

Reply:
yes, but i have a fix now that should be pushed after i add the list of suggestions